policy/modules/system/systemd.te

```diff
@@ -1417,3 +1417,516 @@ optional_policy(`
+# AVC avc:  denied  { setattr } for  pid=1000 comm="systemd-homed" name=".#testuser.identitydw7HTA" dev="dm-9" ino=51127942 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:systemd_homed_var_lib_t:s0 tclass=file permissive=1
+# AVC avc:  denied  { rename } for  pid=1000 comm="systemd-homed" name=".#testuser.identitydw7HTA" dev="dm-9" ino=51127942 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:systemd_homed_var_lib_t:s0 tclass=file permissive=1
+# AVC avc:  denied  { unlink } for  pid=1000 comm="systemd-homed" name="testuser.identity" dev="dm-9" ino=51127939 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:systemd_homed_var_lib_t:s0 tclass=file permissive=1
+manage_files_pattern(systemd_homed_t, systemd_homed_var_lib_t, systemd_homed_var_lib_t)
+
+
+
+#
+# /usr/lib/systemd/systemd-homework create
+# various actions
+#
+
+#### Allow: systemd-homework to mounton /run
+# AVC avc:  denied  { mounton } for  pid=1000 comm="systemd-homewor" path="/run" dev="tmpfs" ino=1 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:var_run_t:s0 tclass=dir permissive=1
+allow systemd_homed_t var_run_t:dir mounton;
+
+#### Allows: systemd-homework to read /dev/random
+# AVC avc:  denied  { open } for  pid=1000 comm="systemd-homewor" path="/dev/random" dev="devtmpfs" ino=8 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:random_device_t:s0 tclass=chr_file permissive=1
+# AVC avc:  denied  { read } for  pid=1000 comm="systemd-homewor" name="random" dev="devtmpfs" ino=8 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:random_device_t:s0 tclass=chr_file permissive=1
+dev_read_rand(systemd_homed_t)
+
+#### Allow: systemd-homework to use tmpfs for various actions
+# AVC avc:  denied  { getattr } for  pid=1000 comm="systemd-homewor" path=2F6D656D66643A646174612D6664202864656C6574656429 dev="tmpfs" ino=2833 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:tmpfs_t:s0 tclass=file permissive=1
+# AVC avc:  denied  { read } for  pid=1000 comm="systemd-homewor" path=2F6D656D66643A646174612D6664202864656C6574656429 dev="tmpfs" ino=2833 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:tmpfs_t:s0 tclass=file permissive=1
+# AVC avc:  denied  { write } for  pid=1000 comm="systemd-homed" path=2F6D656D66643A646174612D6664202864656C6574656429 dev="tmpfs" ino=1783 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:tmpfs_t:s0 tclass=file permissive=1
+# AVC avc:  denied  { ioctl } for  pid=1000 comm="fsck.xfs" path=2F6D656D66643A646174612D6664202864656C6574656429 dev="tmpfs" ino=2835 ioctlcmd=0x5401 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:tmpfs_t:s0 tclass=file permissive=1
+fs_rw_tmpfs_files(systemd_homed_t)
```

Comment:
I would probably create a systemd_home_tmpfs_t type and allow homework to maintain that. You do not see the create here but a type transition to systemd_home_tmpfs_t should work.